prowler/providers/kubernetes/services/core/core_minimize_net_raw_capability_admission/core_minimize_net_raw_capability_admission.py

```diff
@@ -13,10 +13,8 @@ def execute(self) -> Check_Report_Kubernetes:
             report.status = "PASS"
             report.status_extended = f"Pod {pod.name} does not have NET_RAW capability."
             for container in pod.containers.values():
-                if (
-                    container.security_context
-                    and container.security_context.capabilities
-                    and "NET_RAW" in container.security_context.capabilities.add
+                if "NET_RAW" in getattr(
+                    getattr(container.security_context, "capabilities", None), "add", []
```

Comment:
You can use here `attrgetter` like `attrgetter("capabilities.add")(container.security_context)`, it's fine either way.